### ISSUE TITLE
Performance model

### ISSUE THREAD

**Issue by amarquinar** [microsoft/INMT-lite#11]
Hello experts, 

Thank you for your contribution.

I tried to train a model with 90000 sentences from English to Spanish, but the performance of my model is not good.

I tried to change recurrent_hidden to  1024, also, I changed --src_word_vec_size  and --tgt_word_vec_size, as you recomended, but the problem is the same.

Below I show  the results for 20 epochs. As you can see, the validation lost is bad. 
 
Could you help me, please?

      Epoch 10 Batch 0 Loss 5.6960
      Epoch 10 Batch 100 Loss 5.3570
      Epoch 10 Training Loss 5.4667
      Time taken for 1 epoch 529.2065329551697 sec
      
      Epoch 10 Validation Loss 10.6312
      Time taken for validation 60.65908360481262 sec
      
      Epoch 11 Batch 0 Loss 4.7367
      Epoch 11 Batch 100 Loss 5.1832
      Epoch 11 Training Loss 5.1685
      Time taken for 1 epoch 529.9154849052429 sec
      
      Epoch 11 Validation Loss 10.8019
      Time taken for validation 59.725284814834595 sec
      
      Epoch 12 Batch 0 Loss 4.7126
      Epoch 12 Batch 100 Loss 5.1652
      Epoch 12 Training Loss 5.1701
      Time taken for 1 epoch 532.0332324504852 sec
      
      Epoch 12 Validation Loss 11.5809
      Time taken for validation 60.39210081100464 sec
      
      Epoch 13 Batch 0 Loss 4.7674
      Epoch 13 Batch 100 Loss 5.1166
      Epoch 13 Training Loss 5.0376
      Time taken for 1 epoch 527.922877073288 sec
      
      Epoch 13 Validation Loss 10.9454
      Time taken for validation 60.049590826034546 sec
      
      Epoch 14 Batch 0 Loss 4.6097
      Epoch 14 Batch 100 Loss 5.0026
      Epoch 14 Training Loss 4.9551
      Time taken for 1 epoch 533.555011510849 sec
      
      Epoch 14 Validation Loss 11.2711
      Time taken for validation 62.934301137924194 sec
      
      Epoch 15 Batch 0 Loss 4.5907
      Epoch 15 Batch 100 Loss 4.6141
      Epoch 15 Training Loss 4.8234
      Time taken for 1 epoch 530.599461555481 sec
      
      Epoch 15 Validation Loss 11.9423
      Time taken for validation 59.88873314857483 sec
      
      Epoch 16 Batch 0 Loss 4.5649
      Epoch 16 Batch 100 Loss 4.8182
      Epoch 16 Training Loss 4.7378
      Time taken for 1 epoch 533.0982737541199 sec
      
      Epoch 16 Validation Loss 11.9251
      Time taken for validation 62.00506854057312 sec
      
      Epoch 17 Batch 0 Loss 4.2148
      Epoch 17 Batch 100 Loss 4.1748
      Epoch 17 Training Loss 4.5810
      Time taken for 1 epoch 530.5870060920715 sec
      
      Epoch 17 Validation Loss 11.7330
      Time taken for validation 60.24014854431152 sec
      
      Epoch 18 Batch 0 Loss 4.2759
      Epoch 18 Batch 100 Loss 4.6324
      Epoch 18 Training Loss 4.5561
      Time taken for 1 epoch 534.9815211296082 sec
      
      Epoch 18 Validation Loss 12.2905
      Time taken for validation 61.87094283103943 sec
      
      Epoch 19 Batch 0 Loss 4.3838
      Epoch 19 Batch 100 Loss 4.7557
      Epoch 19 Training Loss 4.4805
      Time taken for 1 epoch 533.8275711536407 sec
      
      Epoch 19 Validation Loss 12.4593
      Time taken for validation 62.41018629074097 sec
      
      Epoch 20 Batch 0 Loss 4.0156
      Epoch 20 Batch 100 Loss 4.3589
      Epoch 20 Training Loss 4.3792
      Time taken for 1 epoch 533.2257282733917 sec
      
      Epoch 20 Validation Loss 12.6180
      Time taken for validation 63.08395957946777 sec
